templates/default/solr-env.sh.erb

```diff
@@ -8,7 +8,7 @@ HEARTBEAT_FILE="/data/solr-healthcheck_$opsworks_hostname"
 STARTUP_FILE="$HEARTBEAT_FILE.start"
 
 EXTRA_DISK="/mnt/local_data"
-LOCAL_DIR="$EXTRA_DISK/solr_backup"
+LOCAL_DIR="/var/solr/data/solr_backup"
```

Comment:
can we also move towards removing extra_disk since we have increased the root disk to 100gb.

Reply:
Hmm. We _could_, but I think there is value in keeping them separate, so that if something goes out of control and fills the disk with massive logs or some such thing, it won't crash the whole machine.